molecule/driver/dockerdriver.py

```diff
@@ -223,7 +223,7 @@ def _get_provider(self):
     def _build_ansible_compatible_image(self):
         available_images = [
             tag.encode('utf-8')
-            for image in self._docker.images() for tag in image.get('RepoTags')
+            for image in self._docker.images() for tag in image.get('RepoTags') if tag is not None
```

Comment:
As the maintainer Id prefer the second syntax.  Also, we should have some tests for this.